x-pack/plugins/security_solution/cypress/e2e/cases/connectors.cy.ts

```diff
@@ -86,6 +88,15 @@ describe('Cases connectors', () => {
     });
   });
 
+  after(() => {
+    cy.request({
```

Comment:
NIT: Let's wrap this into a more readable method like `deleteConnector` and place it in `x-pack/plugins/security_solution/cypress/tasks/common.ts` :)

Reply:
Done in https://github.com/elastic/kibana/pull/157254/commits/c046e0a96a53f33786f4853554f0f2a3ef5fa89e